eng/targets/Settings.props

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Authors>dotnet organization</Authors>
+    <Authors>Microsoft Corporation</Authors>
```

```suggestion
    <Authors>Microsoft</Authors>
```

The other `dotnet/*` repos just use 'Microsoft'.